pkg/file/redundancy/level.go

```diff
@@ -165,18 +164,5 @@ func GetReplicaCounts() [5]int {
 // we use an approximation as the successive powers of 2
 var replicaCounts = [5]int{0, 2, 4, 8, 16}
 
-type levelKey struct{}
-
-// SetLevelInContext sets the redundancy level in the context
-func SetLevelInContext(ctx context.Context, level Level) context.Context {
-	return context.WithValue(ctx, levelKey{}, level)
-}
-
-// GetLevelFromContext is a helper function to extract the redundancy level from the context
-func GetLevelFromContext(ctx context.Context) Level {
-	rlevel := PARANOID
-	if val := ctx.Value(levelKey{}); val != nil {
-		rlevel = val.(Level)
-	}
-	return rlevel
-}
+// DefaultLevel is the default redundancy level
+const DefaultLevel = PARANOID
```

Comment:
i wonder if we can find a better name for this 🤔 
this default was used originally for getting the root chunk, maybe the name should reflect that?

Reply:
I think `DefaultLevel` is good enough